libs/components/forms/src/lib/modules/file-attachment/file-attachment.component.html

```diff
@@ -138,6 +138,42 @@
   />
 </div>
 
+<sky-form-errors
+  *ngIf="labelText && (ngControl?.control?.errors || fileErrorValidation)"
+  [id]="errorId"
+  [errors]="
+    ngControl?.errors !== null ? ngControl?.errors : fileErrorValidation
+  "
+  [labelText]="labelText"
+  [showErrors]="ngControl?.touched || ngControl?.dirty"
+>
+  <ng-content select="sky-form-error"></ng-content>
```

Comment:
Since they have no inner content, `ng-content`, `sky-form-error`, etc. can be self-closing, e.g. `<ng-content select="sky-form-error" />`.